release/preview/centos7.9/meta.json

```
@@ -0,0 +1,17 @@
+{
+    "IsLinux" : true,
+    "UseLinuxVersion": false,
+    "PackageFormat": "powershell-${PS_VERSION}-linux-x64.tar.gz",
+    "osVersion": "CentOS 7.9.2009",
+    "tagTemplates": [
+        "#psversion#-centos-#shorttag#-#fulltag#",
+        "preview-centos-#shorttag#"
+    ],
+    "shortTags": [
+        {"Tag": "7.9.2009"}
```

short tag should be 7.9
```suggestion
        {"Tag": "7.9"}
```